Geometry/HGCalCommonData/src/HGCalTileIndex.cc

```diff
@@ -0,0 +1,21 @@
+#include "Geometry/HGCalCommonData/interface/HGCalTileIndex.h"
+
+const int32_t kHGCalLayerOffset = 18;
+const int32_t kHGCalLayerMask = 0x1F;
+const int32_t kHGCalPhiOffset = 0;
+const int32_t kHGCalPhiMask = 0x1FF;
+const int32_t kHGCalRingOffset = 9;
+const int32_t kHGCalRingMask = 0x1FF;
```

Comment:
These should be declared `static` or put in anonymous namespace to avoid polluting the global namespace.